addons/functions/initSettings.sqf

```diff
@@ -10,24 +10,35 @@
+[QGVAR(bigEnvelopeRemovalTime), "SLIDER", localize LSTRING(BigEnvelopeRemovalTime), [localize LSTRING(settingCategory), localize LSTRING(trenchSubCategory)], [5, 600, 40, 0], true] call CBA_fnc_addSetting;
 [QGVAR(allowGiantEnvelope), "CHECKBOX", [localize LSTRING(allowGiantEnvelope_displayName), localize LSTRING(allowGiantEnvelope_tooltip)], [localize LSTRING(settingCategory), localize LSTRING(trenchSubCategory)], true, true] call CBA_fnc_addSetting;
 [QGVAR(giantEnvelopeDigTime), "SLIDER", localize LSTRING(GiantEnvelopeDigTime), [localize LSTRING(settingCategory), localize LSTRING(trenchSubCategory)], [5, 900, 90, 0], true] call CBA_fnc_addSetting;
-[QGVAR(giantEnvelopeRemovalTime), "SLIDER", localize LSTRING(GiantEnvelopeRemovalTime), [localize LSTRING(settingCategory), localize LSTRING(trenchSubCategory)], [-1, 900, -1, 0], true] call CBA_fnc_addSetting;
+[QGVAR(giantEnvelopeRemovalTime), "SLIDER", localize LSTRING(GiantEnvelopeRemovalTime), [localize LSTRING(settingCategory), localize LSTRING(trenchSubCategory)], [5, 900, 90, 0], true] call CBA_fnc_addSetting;
 [QGVAR(allowVehicleEnvelope), "CHECKBOX", [localize LSTRING(allowVehicleEnvelope_displayName), localize LSTRING(allowVehicleEnvelope_tooltip)], [localize LSTRING(settingCategory), localize LSTRING(trenchSubCategory)], true, true] call CBA_fnc_addSetting;
 [QGVAR(vehicleEnvelopeDigTime), "SLIDER", localize LSTRING(VehicleEnvelopeDigTime), [localize LSTRING(settingCategory), localize LSTRING(trenchSubCategory)], [5, 1200, 120, 0], true] call CBA_fnc_addSetting;
-[QGVAR(vehicleEnvelopeRemovalTime), "SLIDER", localize LSTRING(VehicleEnvelopeRemovalTime), [localize LSTRING(settingCategory), localize LSTRING(trenchSubCategory)], [-1, 1200, -1, 0], true] call CBA_fnc_addSetting;
+[QGVAR(vehicleEnvelopeRemovalTime), "SLIDER", localize LSTRING(VehicleEnvelopeRemovalTime), [localize LSTRING(settingCategory), localize LSTRING(trenchSubCategory)], [5, 1200, 120, 0], true] call CBA_fnc_addSetting;
 [QGVAR(allowLongEnvelope), "CHECKBOX", [localize LSTRING(allowLongEnvelope_displayName), localize LSTRING(allowLongEnvelope_tooltip)], [localize LSTRING(settingCategory), localize LSTRING(trenchSubCategory)], true, true] call CBA_fnc_addSetting;
 [QGVAR(LongEnvelopeDigTime), "SLIDER", localize LSTRING(LongEnvelopeDigTime), [localize LSTRING(settingCategory), localize LSTRING(trenchSubCategory)], [5, 1200, 100, 0], true] call CBA_fnc_addSetting;
-[QGVAR(LongEnvelopeRemovalTime), "SLIDER", localize LSTRING(LongEnvelopeRemovalTime), [localize LSTRING(settingCategory), localize LSTRING(trenchSubCategory)], [-1, 1200, -1, 0], true] call CBA_fnc_addSetting;
+[QGVAR(LongEnvelopeRemovalTime), "SLIDER", localize LSTRING(LongEnvelopeRemovalTime), [localize LSTRING(settingCategory), localize LSTRING(trenchSubCategory)], [5, 1200, 100, 0], true] call CBA_fnc_addSetting;
```

Comment:
Das war schon im code berücksichtigt, du fügst damit nur unnötige doppelarbeit hinzu und entfernst backword compatability. Siehe hier:
https://github.com/gruppe-adler/grad_trenches/blob/master/addons/functions/functions/fnc_removeTrench.sqf#L38

Reply:
eli5, raff ich nicht. backward zu was. doppelarbeit für wen.

wenn du sagst presets sollen unterschiedlich sein für dig/removal dann lassma die so 🤔

Reply:
Wenn du mal in den Code schaust, dann ist da ganz klar drin, das -1 bedeutet das es den gleichen wert wie dig hat.

Wenn du jetzt also den Code dazu änderst das es -1 nicht mehr gibt, dann bricht das alle alten missionen.
Und alle änderungen die gruppen an der Dig Time gemacht haben, müssen diese jetzt auch auf die removal time übertragen.